megamek/src/megamek/common/verifier/TestAero.java

```diff
@@ -1018,11 +1014,80 @@ public boolean correctEntity(StringBuffer buff, int ammoTechLvl) {
         correct &= correctControlSystems(buff);
         correct &= !hasIllegalTechLevels(buff, ammoTechLvl);
         correct &= !hasIllegalEquipmentCombinations(buff);
+        correct &= !hasMismatchedLateralWeapons(buff);
         correct &= correctHeatSinks(buff);
         
         return correct;
     }
 
+    /**
```

Comment:
This check is present in the larger aerospace vessels. I added it for use by fixed wing support vehicles, but it also fixes the lack in aerospace and conventional fighters.